database/db.go

```diff
@@ -0,0 +1,40 @@
+package database
+
+import (
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+)
+
+type DB interface {
+	Put(key, value []byte) error
+	Get(key []byte) (value []byte, err error)
+	Close()
+}
+
+type LevelDB struct {
+	db *leveldb.DB
+	wo *opt.WriteOptions
+	ro *opt.ReadOptions
+}
+
+func (db LevelDB) Close() {
+	db.db.Close()
+}
+
+func (db LevelDB) Put(key, value []byte) error {
```

Comment:
in my opnion receiver should be of pointer type

Reply:
LevelDB holds only pointers